### PR TITLE
Refactor auto-beaming tests to use EasyScore.

### PR DESCRIPTION
**Pull request by Silverwolf90**
~~WIP don't merge.~~

EDIT: 

Good to go. 

Removed all clefs since beaming is clef agnostic.

### CODE REVIEW

File: tests/auto_beam_formatting_tests.js [hunk 1]
```diff
@@ -6,1403 +6,962 @@
...
+    simpleAuto(options) {
+      var vf = VF.Test.makeFactory(options);
+      var stave = vf.Stave();
+      var score = vf.EasyScore();
 
-      var voice = new VF.Voice(VF.Test.TIME4_4);
-      voice.addTickables(notes);
+      var voice = score.voice(score.notes(
+        'f5/8, e5, d5, c5/16, c5, d5/8, e5, f5, f5/32, f5, f5, f5'
+      ), { time: '4/4' });
```
Comment (0xfe) [inline tests/auto_beam_formatting_tests.js]: Some of these diff sections are hilariously glorious.

Comment (Silverwolf90) [inline tests/auto_beam_formatting_tests.js]: :D This refactor was a pretty satisfying!

**Comment by Silverwolf90** [comment]
Ok I think this is good to go if the build passes.

I made all the scaling consistent and removed all clefs since beaming is clef agnostic. There are lots of diffs, but all sane I think.

**Comment by 0xfe** [comment]
Diffs look good. And the test code looks way more readable! Thanks for doing this.

**Pull request merged**